### PR TITLE
Blog post to announce linux worker pools for Octopus Cloud

### PR DESCRIPTION
**Pull request by matt-richardson**
# About

This is a post announcing Linux worker pools for Octopus Cloud, as well as Windows 2019 worker pools, a new worker tools image and more.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2020-08/linux-worker-pools-on-octopus-cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+<!-- todo: make sure this is updated once 2020.4.0 is GA on cloud (the worker pool will be auto-created -->
+
+As of today, most customers on Octopus Cloud are on 2020.3.3. This does not yet have a Linux worker pool created by default - this will happen when 2020.4.0 is made available shortly.
+
+In the meantime, go to `Infrastructure -> Worker Pools`, and hit the `Add Worker Pool` button. Selecting `Dynamic` for the Worker Pool Type will use a virtual machine hosted in Octopus Cloud for your deployments.
+
+### Configure an external feed for DockerHub
+
+If you dont yet have a feed setup for DockerHub, go to `Library -> External Feeds` and hit the `Add Feed` button. Select `Docker Container Registry` in the `Feed Type` field, and give it a name - I called mine `Docker Hub`.
```
Comment (benPearce1) [inline blog/2020-08/linux-worker-pools-on-octopus-cloud/index.md:59]:
```suggestion
If you dont yet have a feed setup for DockerHub, go to `Library -> External Feeds` and hit the `Add Feed` button. Select `Docker Container Registry` in the `Feed Type` field, and give it a name - I called mine `Docker Hub` 🥇 .
```

**Review by benPearce1 — APPROVED** [review]
Very nice!

File: blog/2020-08/linux-worker-pools-on-octopus-cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Ubuntu 18.04 workers now available
+
+As of today, I'm happy to announce we now support Ubuntu workers on Octopus Cloud. This is a step forward for us on our journey to not only be the best deployment tool for your Windows workloads but also for your Linux workloads.
+
+![Worker Pool Configuration with Ubuntu 18.04 selected](images/worker-pool-configuration.png)
+
+## Windows 2019 workers now available
+
+I'm also happy to share, we've also launched Windows 2019 workers. We've still in the process of switching everyone over, but you can opt-in to using the new worker images with your instance by navigating to **{{Infrastructure > Worker Pools}}**.
```
Comment (wordlee) [inline blog/2020-08/linux-worker-pools-on-octopus-cloud/index.md:26]: Nobody needs two alsos!
```suggestion
I'm also happy to share, we've launched Windows 2019 workers. We've still in the process of switching everyone over, but you can opt-in to using the new worker images with your instance by navigating to **{{Infrastructure > Worker Pools}}**.
```

File: blog/2020-08/linux-worker-pools-on-octopus-cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+A big positive of switching to the Windows 2019 workers is that Docker is installed as well, so you can make your builds sparkle with your own custom docker images; the only limit is your imagination (oh, and the DockerHub download times).
+
+One thing we're really excited about here is that adding support for Docker really opens up our [execution containers feature](https://octopus.com/blog/execution-containers) which has been behind a feature flag for a while. We'll be removing the feature flag very soon to make it generally available, but you can opt in right now by going to **{{Configuration > Features}}**.
+
+<!-- todo: make sure this :point_up: is updated when feature flag is removed -->
+
+## New worker tools container version
+
+The [worker tools containers](https://hub.docker.com/r/octopusdeploy/worker-tools) have also been updated with the latest tooling ready to unblock lots of goodness, such as the brand new shiny [Terraform 0.13](https://www.hashicorp.com/blog/announcing-hashicorp-terraform-0-13/) version that just got released. Check out [PR #20](https://github.com/OctopusDeploy/WorkerTools/pull/20) for full details of the available versions.
```
Comment (wordlee) [inline blog/2020-08/linux-worker-pools-on-octopus-cloud/index.md:38]: Added back in with a minor tweak:
```suggestion
One last thing, we've updated our [worker tools containers](https://hub.docker.com/r/octopusdeploy/worker-tools) have also been updated with the latest tooling ready to unblock lots of goodness, such as the brand new shiny [Terraform 0.13](https://www.hashicorp.com/blog/announcing-hashicorp-terraform-0-13/) version that just got released. Check out [PR #20](https://github.com/OctopusDeploy/WorkerTools/pull/20) for full details of the available versions.
```

**Pull request merged**